Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -76,6 +76,8 @@ class Outputs:
     add_scores = settings.Setting(False)
     auto_commit = settings.Setting(True)
 
+    pending_selection = settings.ContextSetting(None, schema_only=True)
```

Comment:
I think we have not handled 'schema only' context settings, so setting the flag to True has no effect.